src/Behat/Config/Converter/CustomPrettyPrinter.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Behat\Config\Converter;
+
+use PhpParser\PrettyPrinter\Standard;
+use PhpParser\Node\Expr\Array_;
+use PhpParser\Node\Expr\MethodCall;
+
+class CustomPrettyPrinter extends Standard
+{
+    protected function pExpr_Array(Array_ $node): string
+    {
+        $printedArray = '[';
+        $printedArray .= $this->pCommaSeparatedMultiline($node->items, true);
+
+        return $printedArray . ($this->nl . ']');
+    }
+
+    protected function pExpr_MethodCall(MethodCall $node): string
+    {
+        $result =  $this->pDereferenceLhs($node->var);
```

Comment:
When we do a method call, we indent the call one indent, this makes it much easier to see to which object each method call belongs